suzieq/sqobjects/path.py

```diff
@@ -7,6 +7,7 @@ def __init__(self, **kwargs):
         super().__init__(table='path', **kwargs)
         self._valid_get_args = ['namespace', 'hostname', 'columns',
                                 'vrf', 'source', 'dest']
+        self._unique_def_column = ['hopcount']
```

Comment:
I think hostname makes more sense here than hopCount. The reason is that hopCount is just an increasing number of hops from src to dest. Its not very helpful to get a list of 1, 2, 3 etc. each with how many ever paths there are. So, it'll be something like:
```
0 8
1 8
2 8
```
To indicate that there are 3 hopCounts (0, 1, 2), each with a count of 8